specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2023-05-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,2079 @@
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "snapshot",
+            "in": "query",
+            "description": "Snapshot identifier for the long running operation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OperationDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetOperationStatus": {
+            "$ref": "./examples/GetOperationStatus.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Key": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the key."
+        }
+      }
+    },
+    "KeyValue": {
+      "type": "object",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key of the key-value."
+        },
+        "label": {
+          "type": "string",
+          "description": "The label the key-value belongs to."
+        },
+        "content_type": {
+          "type": "string",
+          "description": "The content type of the value stored within the key-value."
+        },
+        "value": {
+          "type": "string",
+          "description": "The value of the key-value."
+        },
+        "last_modified": {
+          "format": "date-time",
+          "type": "string",
+          "description": "A date representing the last time the key-value was modified."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags of the key-value"
+        },
+        "locked": {
+          "type": "boolean",
+          "description": "Indicates whether the key-value is locked."
+        },
+        "etag": {
+          "type": "string",
+          "description": "A value representing the current state of the resource."
+        }
+      }
+    },
+    "Snapshot": {
+      "type": "object",
+      "required": [
+        "filters"
+      ],
+      "properties": {
+        "name": {
+          "description": "The name of the snapshot.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "description": "The current status of the snapshot.",
+          "type": "string",
+          "enum": [
+            "provisioning",
+            "ready",
+            "archived",
+            "failed"
+          ],
+          "x-nullable": false,
```

Comment:
You should avoid the use of `x-nullable: false`.  It is not needed (it is the default) and just adds noise to the API definition.

Reply:
Removed all occurrences of x-nullable: false